tests/Console/Command/DescribeCommandTest.php

```diff
@@ -32,18 +34,31 @@
  */
 final class DescribeCommandTest extends TestCase
 {
+    /**
+     * @var Application
+     */
+    private $application;
+
+    protected function setUp()
+    {
+        $this->application = new Application();
```

Comment:
> * prevent side effects between tests;

its a private property, side effects can only be occur on purpose

> free memory after all tests;

PHPUnit should release the test class itself, clearing its properties as well. There is no other way of releasing the memory (i.e. setting it to `null` does not help)